Description: moved manpage from level 1 to 8 because memdump is inside /usr/sbin.
Author: Joao Eriberto Mota Filho <eriberto@debian.org>
Last-Update: 2018-11-27
Index: memdump/memdump.1
===================================================================
--- memdump.orig/memdump.1
+++ memdump/memdump.1
@@ -1,4 +1,4 @@
-.TH MEMDUMP 1 
+.TH MEMDUMP 8
 .ad
 .fi
 .SH NAME
